src/ReactEvent.re

```diff
@@ -134,6 +134,36 @@ module Mouse = {
   [@bs.get] external shiftKey: t => bool = "shiftKey";
 };
 
+module Drag = {
+  type tag;
+  type t = synthetic(tag);
+  include MakeEventWithType({
+    type nonrec t = t;
+  });
+
+  // MouseEvent
+  [@bs.get] external altKey: t => bool = "altKey";
+  [@bs.get] external button: t => int = "button";
+  [@bs.get] external buttons: t => int = "buttons";
+  [@bs.get] external clientX: t => int = "clientX";
+  [@bs.get] external clientY: t => int = "clientY";
+  [@bs.get] external ctrlKey: t => bool = "ctrlKey";
+  [@bs.send]
+  external getModifierState: (t, string) => bool = "getModifierState";
+  [@bs.get] external metaKey: t => bool = "metaKey";
+  [@bs.get] external movementX: t => int = "movementX";
+  [@bs.get] external movementY: t => int = "movementY";
+  [@bs.get] external pageX: t => int = "pageX";
+  [@bs.get] external pageY: t => int = "pageY";
+  [@bs.get] [@bs.return nullable]
+  external relatedTarget: t => option(Js.t({..})) = "relatedTarget"; /* Should return Dom.eventTarget */
+  [@bs.get] external screenX: t => int = "screenX";
+  [@bs.get] external screenY: t => int = "screenY";
+  [@bs.get] external shiftKey: t => bool = "shiftKey";
+
+  [@bs.get] external dataTransfer: t => Dom.dataTransfer = "dataTransfer";
```

Comment:
`Dom.dataTransfer` is an opaque type, probably better to return Js.t({..}). Even though it's unsafe it's better to allow users to access by object accessor.